src/vmm/src/signal_handler.rs

```diff
@@ -185,6 +187,12 @@ impl SignalManager {
         // Increment the right metric.
         metric.inc();
 
+        // Never exit on SIGPIPE.
+        if si_signo == SIGPIPE {
```

Comment:
having an `Option` for an exit code would also make this bit of code future proof and a little cleaner

Reply:
Done.